lib/processors/newsprocessor.py

```diff
@@ -0,0 +1,147 @@
+from tornado import gen
+from .lib.utils import process_api_handler
+
+import re
```

Comment:
why did you chose to use `re.search` even though you don't seem to be using any regex grammar?